pipeval/common.py

```diff
@@ -0,0 +1,25 @@
+""" Common functions for PipeVal """
+import os
+
+# pylint: disable=C0103,W0613
+def skippedValidation(name):
+    """
+    Conditionally mark a validation to be skipped
+
+    If the environment variable f'PIPEVAL_SKIP_{name}' is set to `true`,
+    the decorated function will skip that validation.
+    """
+    def decorator(func):
+        return func
+
+    def do_nothing(func):
+        return lambda *args, **kwargs: \
+            print(f'PID:{os.getpid()} - Skipping validation {name.upper()}')
```

Comment:
Strictly speaking, this doesn't do nothing because it prints. I would consider a more accurate name as this method's name does display in debuggers and other analysis tools. Alternatively, you could just return a lambda that returns a lambda. :o)

```python
>>> def skippedValidation(name):
...     return lambda func: lambda *args, **kwargs: print(name)
>>> skippedValidation("foo")(None)()
foo
```

Reply:
True... I updated the name of the function. I added `functools.wraps` to the function run when skipping the call, not sure if this is common practice @nwiltsie @aholmes ?

Reply:
Using `wraps` is common practice. It avoids many possible issues decorators can cause without introducing complexity. It isn't always necessary though.

Reply:
Got it, I'll keep it as is then!